package.json

```diff
@@ -47,27 +40,29 @@
   "homepage": "https://github.com/cfpb/hmda-platform-ui#readme",
   "devDependencies": {
     "babel-jest": "20.0.3",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "7.1.2",
     "babel-plugin-transform-object-rest-spread": "6.19.0",
     "babel-polyfill": "6.16.0",
     "babel-preset-env": "1.6.0",
     "babel-preset-react": "6.16.0",
+    "core-js": "2.5.1",
```

Comment:
Is this being used anywhere?

_While we're at it, I think we can remove `synk`, `mkdirp`, and `minifyify`._